static/css/error.less

```diff
@@ -0,0 +1,70 @@
+/* ==========================================================================
+   cfgov-refresh
+   Page-specific styles for the error pages.
+   ========================================================================== */
+
+.btn__error-home {
+    .respond-to-max(@tablet-min, {
+        width: 100%;
+        text-align: center;
+    });
+
+    .respond-to-max(600px, {
+        .grid_column(12);
+    });
+}
+
+.error_action_container {
+    background-color: @white;
+
+    .respond-to-max(1025px, {
+        .stack-col(content-l_col-1-2);   
+    });
+}
+
+.error_copy_container {
+    .grid_column(6);
+
+    .respond-to-max(600px, {
+        .grid_column(12);
+    });
+}
+
+.error_featured-img {
+    position: absolute;
+    top: 0px;
+    left: 50px;
+    max-width: none;
+
+    &_container {
+        position: relative;
+        .grid_column(6);
+
+        .respond-to-max(600px, {
+            display: none;
+        });
+    }
+
+    &__404 {
+        // Setting top to bottom of content bar. 
+        top: -61px;
```

Comment:
The plug cord shadow is overlapping the green bar at the top:
![screen shot 2015-04-02 at 6 00 04 pm](https://cloud.githubusercontent.com/assets/704760/6974609/23a59f98-d962-11e4-9233-554344b88ce0.png)

I'm wondering if we could avoid this offset altogether and put the image as a background image. I don't think it needs to be an `img` because it's purely decorative and doesn't need to be accessible.

For example, 

```css
.error_main .content_wrapper {
    background: transparent url('/static/img/plug.png') no-repeat top right;
}
```

Then at the tablet/mobile breakpoints the background image could be removed.

Without looking into it closer, I'm not sure if that's the best spot to attach the background image, but that's the general idea.